Specs/DateTime/English/DateTimeModelComplexCalendar.json

```diff
@@ -11824,5 +11824,12 @@
     },
     "NotSupported": "javascript, python",
     "Results": []
+  },
+  {
+    "Input": "ABC-12345-A1B2C3 this is yet to be submitted",
```

Comment:
Can you also add this spec in ExtendedTypes.json as it seems to be the source of the original issue?